### DISCUSSION TITLE
Why don't my assets match up?

### DISCUSSION THREAD

**Discussion by tacastillo** [dagster-io/dagster#17163] [Q&A] [answered]
I have assets made from multiple integrations, such as Fivetran and dbt. The Fivetran tables are the sources of my dbt project. The assets themselves are fine, but my asset with dependencies (such as a dbt asset with source) has a gray box with my source's name pointed to it.

But the name of the asset on the graph is the same as the name of asset in the gray node. How do I connect these two assets together?
![image](https://github.com/dagster-io/dagster/assets/10715711/91095860-935d-47ea-be8a-b5bbbefaed03)
![image](https://github.com/dagster-io/dagster/assets/10715711/27686645-b864-471a-ae27-deea64759ff3)

**Comment by tacastillo** [comment, marked as answer]
In this case, the reason why is because the assets made by fivetran and the dbt source have different asset keys.

At first look, they may be the same, but those assets have prefixes, and the prefixes are likely different.

It's like having two files in two different folders. Just like a file system, you can have multiple files called "README.md", but they would each need to be in different directories. In this case, `_account_netsuite` knows that it needs an `account` asset, but it doesn't know that you have an `account` asset in a different "folder" (or prefix).

So to fix it, you'd have to modify the asset key's prefix on either your fivetran or dbt side.

How to modify the key prefix can vary depending how the assets are made, but in the case of dbt assets and their sources, the easiest way to do this is to write your own [DagsterDbtTranslator](https://docs.dagster.io/_apidocs/libraries/dagster-dbt#dagster_dbt.DagsterDbtTranslator) and write your `get_asset_key` function that names your source's asset keys to match your Fivetran's asset keys. The second example of the docstring will likely fit your needs.

**Reply by slopp** [reply]
For more details, see also: https://github.com/slopp/dagster_dbt_lineage
